build.sbt

```diff
@@ -371,22 +357,27 @@ lazy val zincClassfile = (project in internalPath / "zinc-classfile")
 // re-implementation of scripted engine
 lazy val zincScripted = (project in internalPath / "zinc-scripted")
   .enablePlugins(ContrabandPlugin, JsonCodecPlugin)
-  .disablePlugins(com.typesafe.sbt.SbtScalariform)
   .dependsOn(zinc, zincIvyIntegration % "test->test")
   .settings(
     minimalSettings,
     name := "zinc Scripted",
-    publish := (),
-    publishLocal := (),
+    publish := {},
+    publishLocal := {},
+    skip in publish := true,
```

Comment:
If we already have `skip in publish`, why are `publish` and `publishLocal` definitions still required?